README.md

```diff
@@ -121,6 +121,16 @@ DATABASES = {
 `OPTIONS` is an optional dictionary of parameters that will be passed to
 [`MongoClient`](https://pymongo.readthedocs.io/en/stable/api/pymongo/mongo_client.html).
 
+Alternatively, you can configure Django's `DATABASES` with
```

Comment:
Still missing the motivation as to why the alternative is needed or should be chosen, as well as docs for the kwargs.